harvester/sources/models/anatomy_tool.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from django.contrib.contenttypes.fields import ContentType
+from django.db import models
+from urlobject import URLObject
+
+from datagrowth.configuration import create_config
+from datagrowth.processors import ExtractProcessor
+
+from core.models import HarvestHttpResource
+from anatomy_tool.extraction import AnatomyToolExtraction, ANATOMY_TOOL_EXTRACTION_OBJECTIVE
+# from files.sources.anatomy_tool import AnatomyToolExtraction, ANATOMY_TOOL_EXTRACTION_OBJECTIVE
+
+
+logger = logging.getLogger("harvester")
+
+
+class AnatomyToolOAIPMHManager(models.Manager):
+
+    def extract_seeds(self, latest_update):
```

Comment:
Dit is oude harvester logica en heeft geen relevantie meer voor nieuwe harvester models. De manager op de model kan weer de default worden (dus je hoeft geen objects te definieren)